script.js

```diff
@@ -1,4 +1,261 @@
+      return `${userName} threw ${input} and computer threw ${randomComputershand}. You win!<br><br>Your score: ${myScore}<br><br> Your opponent's score: ${computersScore}<br><br>Number of draws: ${numberOfDraws}<br><br>Total number of rounds: ${totalNumberOfRounds}<br><br>Your winning rate: ${myWinningPercentage}%<br><br>The computer's winning rate: ${computersWinningPercentage}%`;
+    }
+
+    if (
+      (input == "paper" && randomComputershand == "stone") ||
+      (input == "reversed scissors" && randomComputershand == "stone")
+    ) {
+      myScore = myScore + 1;
+      computersScore = computersScore + 0;
+      numberOfDraws = numberOfDraws + 0;
+      totalNumberOfRounds = myScore + computersScore + numberOfDraws;
+      myWinningPercentage = (myScore / totalNumberOfRounds) * 100;
+      computersWinningPercentage = 100 - myWinningPercentage;
+      return `${userName} threw ${input} and computer threw ${randomComputershand}. You win!<br><br>Your score: ${myScore}<br><br> Your opponent's score: ${computersScore}<br><br>Number of draws: ${numberOfDraws}<br><br>Total number of rounds: ${totalNumberOfRounds}<br><br>Your winning rate: ${myWinningPercentage}%<br><br>The computer's winning rate: ${computersWinningPercentage}%`;
+    }
+
+    if (
+      (input == "stone" && randomComputershand == "scissors") ||
+      (input == "reversed paper" && randomComputershand == "scissors")
+    ) {
+      myScore = myScore + 1;
+      computersScore = computersScore + 0;
+      numberOfDraws = numberOfDraws + 0;
+      totalNumberOfRounds = myScore + computersScore + numberOfDraws;
+      myWinningPercentage = (myScore / totalNumberOfRounds) * 100;
+      computersWinningPercentage = 100 - myWinningPercentage;
+      return `${userName} threw ${input} and computer threw ${randomComputershand}. You win!<br><br>Your score: ${myScore}<br><br> Your opponent's score: ${computersScore}<br><br>Number of draws: ${numberOfDraws}<br><br>Total number of rounds: ${totalNumberOfRounds}<br><br>Your winning rate: ${myWinningPercentage}%<br><br>The computer's winning rate: ${computersWinningPercentage}%`;
+    }
+
+    // All draw cases:
+    if (input == randomComputershand) {
+      myScore = myScore + 0;
+      computersScore = computersScore + 0;
+      numberOfDraws = numberOfDraws + 1;
+      totalNumberOfRounds = myScore + computersScore + numberOfDraws;
+      myWinningPercentage = (myScore / totalNumberOfRounds) * 100;
+      computersWinningPercentage = 100 - myWinningPercentage;
+      return `${userName} threw ${input} and computer threw ${randomComputershand}. It's a draw!<br><br>Your score: ${myScore}<br><br> Your opponent's score: ${computersScore}<br><br>Number of draws: ${numberOfDraws}<br><br>Total number of rounds: ${totalNumberOfRounds}<br><br>Your winning rate: ${myWinningPercentage}%<br><br>The computer's winning rate: ${computersWinningPercentage}%`;
+    }
+
+    if (input == "reversed scissors" && randomComputershand == "scissors") {
```

Comment:
maybe you can use else if here since only one of these scenarios will be fulfilled at any given round. By using else if, it will prevent the subsequent code from being ran if the if condition was met.